src/node/cli.ts

```diff
@@ -556,6 +562,10 @@ export async function setDefaults(cliArgs: UserProvidedArgs, configArgs?: Config
   const proxyDomains = new Set((args["proxy-domain"] || []).map((d) => d.replace(/^\*\./, "")))
   args["proxy-domain"] = Array.from(proxyDomains)
 
+  if (!args["disable-file-downloads"]) {
```

Comment:
❓ I wasn't sure where to put this. Happy to move if it should be set somewhere else.

Reply:
I think we can leave this out, for boolean CLI flags we assume not being set is the same as false.

Reply:
Hmm...does that mean I need to update the tests then? My thinking was if it's not passed, we need to set this to `false`

But we don't do this for `help` which is another boolean flag with default set to `false`. Do I need to set it in `toCodeArgs` then?

Reply:
Yup, you would need to update the tests.  We should not need it in `toCodeArgs`, we can do the same as with `disable-update-check`.  `disable-file-downloads` already allows undefined in `serverEnvironmentService` so no need to do anything.

Reply:
fwiw I think undefined semantically makes more sense than false because the user did not provide it (rather than them explicitly providing false).